presto-hive/src/main/java/com/facebook/presto/hive/parquet/reader/ParquetColumnChunk.java

```diff
@@ -33,22 +31,22 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static com.facebook.presto.hive.parquet.ParquetTypeUtils.getParquetEncoding;
+import static io.airlift.slice.Slices.wrappedBuffer;
+
 public class ParquetColumnChunk
         extends ByteArrayInputStream
```

Comment:
I think instead of inheritance composition makes more sense here, what do you think? A `ParquetColumnChunk` is not really a `ByteArrayInputStream`.